rt_bi_emulator:
  av1:
    ros__parameters:
      robotId: 1
      updateInterval: 0.75
      timesSecs:
        - 0.0
        - 50.0
      cutOffTime: inf
      saPoses:
        - "[120.804, 134.190, -13.0]"
        - "[120.804, 134.190, -53.0]"
      centersOfRotation:
        - "[120.804, 134.190]"
        - "[120.804, 134.190]"
      fovs:
        - "[[283.602, 65.354], [296.297, 119.257], [120.804, 134.190]]"
        - "[[200.612, -24.140], [244.985, 8.992], [120.804, 134.190]]"
